src/sdl/i_system.c

```diff
@@ -104,6 +104,12 @@ typedef LPVOID (WINAPI *p_MapViewOfFile) (HANDLE, DWORD, DWORD, DWORD, SIZE_T);
 #endif
 #endif
 
+#if (defined (__unix__) && !defined (_MSDOS)) || (defined (UNIXCOMMON) && !defined(__APPLE__))
+#include <errno.h>
+#include <sys/wait.h>
+#define NEWSIGNALHANDLER
```

Comment:
We should exclude Haiku from this, since Haiku has a built-in crash dialog that are orders of magnitude more valuable than what SRB2 can provide (you can attach a debugger directly to the crashed process, for example).